target/linux/mediatek/image/make_bpi-r2_bundle_image.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2013 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
```

Comment:
this file should be turned into a real image building code template and not a shell script from within the image code.

Reply:
How would one do that? Maybe there are examples in the code tree that I can use?